src/sagemaker/workflow/steps.py

```diff
@@ -160,9 +160,6 @@ def arguments(self) -> RequestType:
         NOTE: The CreateTrainingJob request is not quite the args list that workflow needs.
         The TrainingJobName and ExperimentConfig attributes cannot be included.
         """
-        self.estimator.disable_profiler = True
-        self.estimator.profiler_config = None
-        self.estimator.profiler_rules = None
```

Comment:
Are there any edge cases that can arise by having it enabled by default?

Reply:
I don't think so. We enable the profiler by default if a user launch a training job directly by doing `estimator.fit()`. We could do the same for pipeline users.

Reply:
Can we add an integ test?

Reply:
Yes. Updated the existing test to verify the profiler config.

Reply:
Awesome! Does this work in every region?

Reply:
Yes. Profiler config is not available in CN, or GovCloud, pipeline is not available in those regions either.